src/components/CategoryNavContainer/CategoryNav/CategoryNav.js

```diff
@@ -23,30 +23,32 @@ class CategoryNav extends Component {
 
   render() {
     return (
```

Comment:
All I did here was add a `<div className={style.wrapper}>` wrapper, but git got confused so it looks like I changed a whole lotta stuff.